src/constants.ts

```diff
@@ -6,13 +6,16 @@ export enum Inputs {
   Name = 'name',
   Tag = 'tag',
   Source = 'source',
-  Workflow_name = 'workflow-name'
+  Workflow_name = 'workflow-name',
+  Key = 'key',
+  Value = 'value'
```

Comment:
Doing a cit shouldn't cost much, and can be really interesting like:
```yaml
args:
    key: value
    key2: value
```

Reply:
As per my statement above https://github.com/scality/action-artifacts/pull/119#discussion_r868340602

As much as I would like to use full yaml specs on this one, I don't think it's a good choice (and the input will most likely be given as a string)
I can suggest doing something like this:
```
args: |
  key=value
```
For the sake of consistency between other actions available elsewhere.

Reply:
Good for me, it keep the pair key, value  super clear